### ISSUE TITLE
Broken link in README to 'install-info'

### ISSUE THREAD

**Issue by jbampton** [dimitri/el-get#2799]
Broken link is:

https://developer.apple.com/library/archive/documentation/Darwin/Reference/ManPages/man1/install-info.1.html

<img width="829" alt="Screen Shot 2021-01-18 at 12 16 38 am" src="https://user-images.githubusercontent.com/418747/104845766-e3c48780-5922-11eb-8258-e078b03ffa44.png">

**Comment by jbampton** [comment]
Error on the Debian site from this link:

<img width="556" alt="Screen Shot 2021-01-18 at 12 21 23 am" src="https://user-images.githubusercontent.com/418747/104845881-6fd6af00-5923-11eb-82fe-5012e1c4b704.png">

<img width="844" alt="Screen Shot 2021-01-18 at 12 21 44 am" src="https://user-images.githubusercontent.com/418747/104845864-5cc3df00-5923-11eb-9864-29677a0ee114.png">

**Comment by stephan-cr** [comment]
I'm not a Mac user and I'm not sure if `install-info` is supposed to be installed by default, but it might be provided by https://formulae.brew.sh/formula/texinfo.

**Comment by iainhouston** [comment]
Yes, I can confirm that `brew install texinfo` will install a working `/opt/homebrew/bin/install-info`
